spec/factories/users.rb

```diff
@@ -38,12 +38,14 @@
 
   factory :mhv_user, class: 'User' do
     uuid 'b2fab2b5-6af0-45e1-a9e2-394347af91ef'
-    mhv_last_signed_in Time.current
-    email 'abraham.lincoln@vets.gov'
-    first_name 'abraham'
-    last_name 'lincoln'
+    mhv_last_signed_in { Faker::Time.between(1.week.ago, 1.minute.ago, :all) }
```

Comment:
Nothing in particular. I prefer using Faker for creating factory data over hardcoding specific values because Faker randomizes the data in a way that can help detect failures that don't occur with the hard values. Faker was already included in the project, and I needed to tweak this factory anyway so I figured I'd improve it while I was at it.